package.json

```diff
@@ -25,6 +25,7 @@
     "@codemirror/lang-sql": "^6.9.0",
     "@electron-toolkit/preload": "^3.0.1",
     "@electron-toolkit/utils": "^4.0.0",
+    "@google/generative-ai": "^0.24.1",
```

Comment:
it looks like this package was deprecated:

https://www.npmjs.com/package/@google/generative-ai